File: python/1426.counting-elements.py
class Solution:
    def countElements(self, arr: List[int]) -> int:
        freq = {}
        for item in arr:
            freq[item] = freq.get(item,0)+1
        res = 0
        for (val, count) in freq.items():
            if (val+1) in freq:
                res += count
        return res

File: cpp/1426.counting-elements.cpp
class Solution {
public:
    int solve(vector<int>& arr) {
	    unordered_map<int, int>freq;
	    for(auto num:arr)freq[num]++;
	    int res = 0;
	    for(auto [val, cont]:freq){
		  if(freq.count(val+1)) res+=cont;
	    }
	    return res;
    }
    int countElements(vector<int>& arr) {
	return solve(arr);
    }
};
